com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/Exposure.compute

```diff
@@ -12,6 +12,7 @@ TEXTURE2D(_InputTexture);
 
 #define PREPASS_TEX_SIZE 1024.0
 #define PREPASS_TEX_HALF_SIZE 512.0
+#pragma enable_d3d11_debug_symbols
```

Comment:
please comment this line

Reply:
Argh yes, didn't see it first time around, doing now.